### PR TITLE
[12.0][MIG] purchase_order_analytic_search

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
The search works in the purchase lines but not in the purchase order

File: purchase_order_analytic_search/models/purchase_order.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                      ('account_analytic_id', 'in', value)]
+        else:
+            if isinstance(value, int):
+                # we are searching on the id of the analytic_account
+                domain = [('order_id', '!=', False),
+                          ('account_analytic_id', '=', value)]
+            else:
+                # assume we are searching on the analytic account name
+                domain = [('order_id', '!=', False),
+                          ('account_analytic_id.name', 'like', value),
```
Comment (AaronHForgeFlow) [inline purchase_order_analytic_search/models/purchase_order.py]: ilike to be case-sensitive as standard odoo ? Not sure

Comment (MiquelRForgeFlow) [reply purchase_order_analytic_search/models/purchase_order.py]: done

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional + code review

File: purchase_order_analytic_search/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2015-17 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Purchase Order Analytic Search",
+    "summary": """Search purchase orders by analytic account. New menu entry in
+                Purchasing to list purchase order lines.""",
+    "version": "12.0.1.0.0",
+    "website": "https://odoo-community.org/",
```
Comment (nikul-serpentcs) [inline purchase_order_analytic_search/__manifest__.py]: https://github.com/OCA/purchase-workflow

**Review by nikul-serpentcs — COMMENTED** [review]
minor change

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Pull request merged**